adagrams/game.py

```diff
@@ -1,11 +1,142 @@
+import random
+import copy
+
+ # Two Dimensional Array
+
+LETTER_POOL = [
+	['A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A'],
+	['B', 'B'], 
+	['C', 'C'], 
+	['D','D','D','D'],
+	['E','E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E'], 
+	['F', 'F'], 
+	['G', 'G', 'G'], 
+	['H', 'H'], 
+	['I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I'], 
+	['J'], 
+	['K'], 
+	['L', 'L', 'L', 'L'], 
+	['M', 'M'],
+	['N', 'N', 'N', 'N', 'N', 'N'],
+	['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O'], 
+	['P', 'P'], 
+	['Q'], 
+	['R', 'R', 'R', 'R', 'R', 'R'], 
+	['S', 'S', 'S', 'S'], 
+	['T', 'T', 'T', 'T', 'T', 'T'], 
+	['U', 'U', 'U', 'U'], 
+	['V', 'V'], 
+	['W', 'W'], 
+	['X'], 
+	['Y', 'Y'],
+	['Z']
+]
+
+# helper function to create one big list of strings
+
+def get_one_list_of_letters():
+
+    total_letters_list = []
+    for elem in LETTER_POOL:
+        for char in elem:
+            total_letters_list.append(char)
+    return total_letters_list
+
 def draw_letters():
-    pass
+
+    total_letters_list = get_one_list_of_letters()
+
+    # start with an empty hand and initialize counter at 0
+    hand = []
+    i = 0
+    while i < 10:
+        # randomize letter choice
+        random_letter = total_letters_list[random.randint(0, 97)]
+        # ensure that random letter choice in hand is never greater than the max amount in the letter pool
+        # if less than the max amount then append to empty list which is our hand of letters
+        if hand.count(random_letter) < total_letters_list.count(random_letter):
+            hand.append(random_letter)
+        else:
+            # use continue to end this iteration and immediately start next iteration
+            continue
+        i += 1
```

Comment:
`count` is a great function to use in a situation like this! That way you don't have worry about manipulating any data and removing things! An alternative refactor we could do to make the code a bit more concise is this:
```python
while len(hand) < 10:
     random_letter = total_letters_list[random.randint(0, 97)]

     if hand.count(random_letter) < total_letters_list.count(random_letter):
            hand.append(random_letter)
```

This code will run the same as yours, this code just doesn't keep track of a counter since the times we `append` to `hand` is equal to `len(hand)`